dataland-frontend/tests/e2e/support/TestUtility.ts

```diff
@@ -9,7 +9,7 @@ export type DataEnvironment = "fakeFixtures" | "realData";
 
 export function describeIf(name: string, execConfig: ExecutionConfig, fn: (this: Suite) => void): Suite {
   const executionEnvironment = Cypress.env("EXECUTION_ENVIRONMENT") as ExecutionEnvironment;
-  const dataEnvironment = Cypress.env("DATA_ENVIRONMENT") as DataEnvironment;
+  const dataEnvironment = Cypress.env("data_environment") as DataEnvironment;
```

Comment:
Warum wurde das hier geändert?

Reply:
Um es konsistent zu halten (siehe die Cypress-env ```commit_id```) , habe ich Cpyress-envs (die sich ja von den Operating System envs unterscheiden), die innerhalb der config erzeugt werden, und zwar hier:

```Typescript
export default defineConfig({
    env: {
        commit_id: require("git-commit-id")({cwd: "../"}),
        data_environment: getDataEnvironmentBasedOnOperatingSystemEnv()
    },
```

in small case / snake case geschrieben.

Die Cypress-env EXECUTION_ENVIRONMENT wird jedoch von den Container-Envs des Test-Containers eingelesen, und ist deshalb groß geschrieben.

Wir können aber gerne nochmal über eine passende Konvention hier sprechen bzw. ich habe keine starke Meinung hierzu.

Reply:
das ergibt dann Sinn so